src/main/java/org/gitlab4j/api/MergeRequestApi.java

```diff
@@ -577,4 +578,34 @@ public MergeRequest unapproveMergeRequest(Integer projectId, Integer mergeReques
         Response response = post(Response.Status.OK, (Form)null, "projects", projectId, "merge_requests", mergeRequestIid, "unapprove");
         return (response.readEntity(MergeRequest.class));
     }
+
+    /**
+     * Get merge request with changes information.
+     *
+     * GET /projects/:id/merge_requests/:merge_request_iid/changes
+     *
+     * @param projectId the project ID to get the merge requests for
+     * @param mergeRequestIid the IID of the merge request to get
+     * @return a merge request including its changes
+     * @throws GitLabApiException if any exception occurs
+     */
+    public MergeRequest getMergeRequestChanges(Integer projectId, Integer mergeRequestIid) throws GitLabApiException {
+        Response response = get(Response.Status.OK, null, "projects", projectId, "merge_requests", mergeRequestIid, "changes");
+        return (response.readEntity(MergeRequest.class));
+    }
+    
+    /**
+     * Get participants of merge request.
+     * 
+     * GET /projects/:id/merge_requests/:merge_request_iid/participants
+     * 
+     * @param projectId the project ID to get the merge requests for
+     * @param mergeRequestIid the IID of the merge request to get
+     * @param itemsPerPage the number of Participant instances that will be fetched per page
+     * @return all participants for the specified merge request
+     * @throws GitLabApiException if any exception occurs
+     */
+    public Pager<Participant> getParticipants(Integer projectId, Integer mergeRequestIid, int itemsPerPage) throws GitLabApiException {
```

Comment:
Should be returning `Pager<User>`  no need to create a new class for the results.